### DISCUSSION TITLE
Project level custom variable

### DISCUSSION THREAD

**Discussion by mussels** [opengisch/QField#4374] [Q&A]
I set up a project variable within a project crated with QGIS Desktop (Project -> Properties -> Variables, Project). To be clear, there are two sets of variables here: Global (i.e. application level) and Project (i.e. project level).
The newly created variable is successfully converted to a QField variable. I know this because a vector layer I setup uses this variable to populate a field value automatically. It works but the problem is that it is not available to the user.
The user must be able to edit this variable, presumably through QField settings, when the project is open.  But the variable, although existing, it doesn’t seem to be accessible to the user (it is obviously accessible to the application/project).
The variables section of the settings in QField only shows the global variables (the same set of Global variables in QGIS Desktop) but the set of project variables seems missing. Creating a variable here of the same name, in an attempt to overwrite the project variable’s value, does not work, and I suspect this is because the “hidden” project level variable scope comes first.
Am I right thinking the project variable was accessible by the user with previous versions of QField? Am I perhaps missing something obvious?

QGIS Desktop 3.22
QField 2.6.0 - Windows 11 
QField 2.8.2 – IpadOS 16.3

**Comment by soester** [comment]
https://github.com/opengisch/QField/issues/1814
